src/editors/containers/TextEditor/hooks.test.jsx

```diff
@@ -83,6 +117,33 @@ describe('TextEditor hooks', () => {
         output = module.editorConfig(newprops);
         expect(output.initialValue).toBe(htmltext);
       });
+      test('It configures plugins and toolbars correctly', () => {
+        output = module.editorConfig(props);
+        expect(output.init.plugins).toBe(module.getConfig('plugins'));
```

Comment:
Would it be better to do: 
for each value in module.pluginConfig.plugins:
   expect(output.init.plugins.contains(that value)).toBe(true);

Reply:
I did that :0